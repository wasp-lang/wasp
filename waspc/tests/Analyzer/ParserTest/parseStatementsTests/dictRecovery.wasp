app todoApp {
  foo: ("a"),
  bar: ("b")
}
