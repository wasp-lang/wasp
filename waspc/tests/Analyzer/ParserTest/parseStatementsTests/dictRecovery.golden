Parse error at 2:8-12:
  Tuple only contains 1 values, but it must contain at least 2 values

Parse error at 3:8-12:
  Tuple only contains 1 values, but it must contain at least 2 values
